web_switch_company_warning/static/src/js/switch_company_warning.js

```diff
@@ -0,0 +1,54 @@
+odoo.define('web_switch_company_warning.widget', function (require) {
+    'use strict';
+
+    var Widget = require('web.Widget');
+    var UserMenu = require('web.UserMenu');
+    var session = require('web.session');
+    //Show a big banner in the top of the page if the company has been
+    //changed in another tab or window (in the same browser)
+
+    if (!window.SharedWorker) {
+        //not supported
+        return;
+    }
+    var SwitchCompanyWarningWidget = Widget.extend({
+        template:'web_switch_company_warning.warningWidget',
+        init: function() {
+            this._super();
+            var self = this;
+            var w = new SharedWorker('/web_switch_company_warning/static/src/js/switch_company_warning_worker.js');
```

Comment:
W7903(javascript-lint), ] Line 19 exceeds the maximum line length of 80. [Error/max-len]